### PR TITLE
Additional PEM helpers

### PR DESCRIPTION
**Pull request by yschimke**
Splitting out from https://github.com/cashapp/certifikit/pull/94

### CODE REVIEW

**Comment by yschimke** [comment]
@swankjesse request post review.

File: certifikit-cli/src/jvmMain/kotlin/app/cash/certifikit/cli/certs.kt [hunk 1]
```diff
@@ -62,3 +62,15 @@ internal suspend fun X509Certificate.writePem(
     }
   }
 }
+
+@Suppress("BlockingMethodInNonBlockingContext")
```
Comment (oldergod) [inline certifikit-cli/src/jvmMain/kotlin/app/cash/certifikit/cli/certs.kt:66]: I see other methods have it but none of them actually need it I think?

Comment (oldergod) [reply certifikit-cli/src/jvmMain/kotlin/app/cash/certifikit/cli/certs.kt:66]: I removed them and I don't see anything.
I'll push a PR

**Review by oldergod — COMMENTED** [review]
LGTM

**Pull request merged**